pkg/apis/certmanager/v1alpha2/generic_issuer.go

```diff
@@ -23,6 +23,7 @@ import (
 	cmacme "github.com/jetstack/cert-manager/pkg/apis/acme/v1alpha2"
 )
 
+// GenericIssuer is ...
```

Comment:
I really need help w/ types

(Also, the fact that I get to do this for multiple versions of the API is not fun.)